lnst/Recipes/ENRT/VlansOverBondRecipe.py

```diff
@@ -200,6 +192,12 @@ def generate_test_wide_description(self, config: EnrtConfiguration):
                 host1.bond0.miimon
             )
         ]
+
+        if self.params.bonding_mode in ["active-backup", "1"]:
+            desc += ["Configured {}.{}.fail_over_mac = {}".format(
+                host1.hostid, host1.bond0.name,
+                host1.bond0.fail_over_mac
+            )]
```

Comment:
I think this shoud go into  the BondingMixin class. It's duplicated over individual recipes and it makes more sense to generate this within the BondingMixin class.

Reply:
Fixed.